cli/magic-config.ts

```diff
@@ -1108,7 +1107,9 @@ async function processCreateOptions(options: any): Promise<void> {
     };
   }
```

Comment:
For now remove the `if` and always add the cross encoder model to the configuration. The `crossEncodingEnabled` setting is already defining if cross encoders are enabled or not. This should be consistent on how we handle embedding models. A possible alternative in both cases would be to put an empty list in case they are disabled.